packages/tools/client-debugger/client-debugger-view/src/components/ContainerDevtoolsView.tsx

```diff
@@ -155,10 +160,17 @@ function _ContainerDevtoolsView(props: _ContainerDevtoolsViewProps): React.React
 			</Stack.Item>
 			<Stack.Item style={{ width: "100%", height: "100%", overflowY: "auto" }}>
 				<Stack tokens={{ childrenGap: 10 }}>
-					<PanelViewSelectionMenu
-						currentSelection={innerViewSelection}
-						updateSelection={setInnerViewSelection}
-					/>
+					<TabList selectedValue={innerViewSelection} onTabSelect={onTabSelect}>
```

Comment:
The previous implementation was able to generate new tabs dynamically if we added a new flag to the `PanelView` enum. Could we update this logic to preserve that? I.e., generate the list of tab options from the enum flags?